packages/arb-token-bridge-ui/package.json

```diff
@@ -89,13 +89,13 @@
     "@types/react-dom": "^18.2.4",
     "@types/react-virtualized": "^9.21.20",
     "@typescript-eslint/eslint-plugin": "^5.57.0",
-    "@typescript-eslint/parser": "^5.44.0",
+    "@typescript-eslint/parser": "^6.13.2",
```

Comment:
remove a warning for unsupported version of a subpackage when running `yarn lint` locally